Miniscript-Template-002-5-Time-Layered-Multisig.mediawiki

```diff
@@ -0,0 +1,76 @@
+<pre>
+  Miniscript Template: 2
```

Comment:
I recommend the front matter be

MSTemplate: 2

similar to the no space file naming...